src/main/java/org/myrobotlab/framework/repo/ServiceData.java

```diff
@@ -53,7 +53,7 @@ public class ServiceData implements Serializable {
 
   private static final long serialVersionUID = 1L;
 
-  static private String serviceDataCacheFileName = FileIO.getCfgDir() + File.separator + "serviceData.json";
+  static private String serviceDataCacheFileName = "libraries" + File.separator + "serviceData.json";

```

Comment:
There should probably be a constant somewhere to store the string "libraries", don't want a typo to cause problems

Reply:
fixed